python/pyarrow/tests/test_cuda_numba_interop.py

```diff
@@ -75,6 +75,89 @@ def make_random_buffer(size, target='host', dtype='uint8', ctx=None):
     raise ValueError('invalid target value')
 
 
+@pytest.mark.parametrize("c", range(len(context_choice_ids)),
+                         ids=context_choice_ids)
+@pytest.mark.parametrize("dtype", dtypes, ids=dtypes)
+@pytest.mark.parametrize("size", [0, 1, 8, 1000])
```

Comment:
There is no reason to make tests slower by testing N different lengths. Just test a zero length and a non-zero length.

Reply:
Will remove the `size == 8` case.

Reply:
The speedup from removing tests is really marginal.

Reply:
I see, thanks.